config/environment.rb

```diff
@@ -1,4 +1,5 @@
+require 'pry'
 require 'bundler'
 Bundler.require
 
-require_all 'lib'
+require_all 'lib'
```

Comment:
It is a [Ruby style convention](https://github.com/styleguide/ruby) to end every file with a blank newline.